web/src/Setting.js

```diff
@@ -1592,6 +1592,16 @@ export function getProviderSubTypeOptions(category, type) {
     } else {
       return [];
     }
+  } else if (category === "Blockchain") {
+    if (type === "ChainMaker") {
+      return [
+        {id: "permissionedwithcert", name: "permissionedwithcert"},
```

Comment:
I don't think this is a "sub type", it's just an option. And I think this option can be hard-coded for now. Because `permissionedwithcert` is the only one that we support for now, right?